docs/source/models/gbd2017/risk_attributable_causes/iron_deficiency/index.rst

```diff
@@ -4,5 +4,152 @@
+ chronic blood loss, altered red blood cell morphology, inadequate nutritional 
+ intake, and others.
+
+**Iron deficiency anemia** is a type of anemia that is due to insufficient 
+iron levels, which lead to a deficiency of hemoglobin in the blood. Notably, 
+iron deficiency anemia can occur when dietary intake of iron is insufficient, 
+although it may occur in other situations as well, such as when iron is lost 
+through bleeding (ex: menstrual disorders, hookworm disease, etc.). Iron 
+deficiency anemia is the most common cause of anemia globally in most 
+populations.
+
+**Dietary iron deficiency anemia** is a specific type of iron deficiency anemia 
+that is due to inadequate dietary intake of iron, leading to inadequate iron 
+levels in the body and a subsequent deficiency of hemoglobin in the blood.
+
+Modeling Iron Deficiency in GBD 2017
+------------------------------------
+
+In GBD 2017, the cause dietary iron deficiency is a population attributable 
+fraction (PAF) of 1 cause with the iron deficiency risk factor. This means 
+that 100% of the dietary iron deficiency cases are attributable to the iron 
+deficiency risk factor. Notably, there are additional causes other than 
+dietary iron deficiency that are attributable to the iron deficiency risk 
+factors (ex: maternal disorders).
+
+Additionally, there is an anemia *impairment* modeled in GBD 2017 that 
+represents **all** forms of anemia that are attributable to several causes, 
+including causes such as hemoglobinopathies and hemolytic anemias that are not 
+causally attributable to iron deficiency. 
+
+Iron Deficiency Risk Factor
++++++++++++++++++++++++++++
+
+The iron deficiency risk factor in GBD 2017 is modeled as a distribution of 
+hemoglobin concentrations. The risk factor is a **continuous measure**.
+
+.. todo::
+
+	Describe the population and distribution type (normal vs. ensemble)
+
+Modeling Strategy for the Iron Deficiency Risk Factor
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
 .. todo::
-   Describe PAF-of-1 cause model
+
+	Describe anemia envelope and causal attribution
+
+Risk Factor Hierarchy
+^^^^^^^^^^^^^^^^^^^^^
+
+.. image:: iron_risk_hierarchy.svg
+
+Dietary Iron Deficiency Cause
++++++++++++++++++++++++++++++
+
+The dietary iron deficiency cause in GBD 2017 that is 100% attributable to the 
+iron deficiency risk factor. The dietary iron deficiency cause in GBD is a 
+YLD-only cause, meaning that it contributes to morbidity, but not mortality.
+
+Modeling Strategy for the Dietary Iron Deficiency Cause
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+The dietary iron deficiency cause in GBD 2017 is not modeled directly. Rather, 
+the dietary iron deficiency cause is estimated as the remaining cases of 
+anemia after all of the other anemic cases in the overall anemia envelope were 
+causally attributed to their respective attributable causes (ex: 
+hemoglobinopathies and hemolytic anemias).
```

Comment:
Even though these are risk-attributable, I think it is the case that the cause model is fully responsible for the YLDs, and then they are retroactively attributed to the risk exposure. (though it might be slightly more complicated than that). Thus it is easier for me to process/understand, i think, if this section comes before the risk factor section.

I think it would also be helpful to have a high-level understanding of how the parent envelope is modeled, and maybe at what stage they subtract out the other child models?